ci/terraform/oidc/ipv-handoff-alerts.tf

```diff
@@ -51,24 +29,6 @@ resource "aws_cloudwatch_metric_alarm" "ipv_handoff_p1_cloudwatch_alarm" {
   alarm_description   = "${var.ipv_p1_alarm_error_threshold} or more IPV handoff errors have occurred in ${var.environment}.ACCOUNT: ${data.aws_iam_account_alias.current.account_alias}"
   alarm_actions       = [var.environment == "production" ? data.aws_sns_topic.pagerduty_p1_alerts[0].arn : data.aws_sns_topic.slack_events.arn]
 
-  metric_query {
-    id          = "combined_error_count"
-    label       = "ipv-handoff-pre-and-post-split-combined-error-count"
-    expression  = "ipv_authorize_lambda_error_count+authentication_callback_lambda_error_count"
```

Comment:
If you look at the PR that added this change, the end state we want is the same as before this change, but for the authentication callback lambda instead of the ipv authorize lambda